### ISSUE TITLE
(Mac) Does not work correctly when DAW is set to certain sample rates

### ISSUE THREAD

**Issue by tertiumsquid** [GuitarML/Proteus#11]
The plugin seems to work perfectly when the sample rate is set to 44,100hz, but on some sample rates it does not sound right.  Depending on the model being used, 48k sometimes works, but 96k and 192k rarely do.  Multiples of 44,100hz (like 88,200hz) sometimes work, but sometimes don't.

Tested on two different machines (one Intel and one ARM) with 3 different audio interfaces (Focusrite 18i20mk3, Presonus 1818VSL, Resident Audio T4), using both Logic and Reaper (latest versions).

**Comment by GuitarML** [comment]
@tertiumsquid Yes, the models were trained using 44100 data, there is an internal samplerate converter but it's too slow for higher samplerates. I would like to fix this, but in the meantime you should be able to use the GuitarML module in [BYOD](https://github.com/Chowdhury-DSP/BYOD) to run Proteus models at the higher samplerates. ChowDSP has an updated samplerate handler for neural models. 

Thanks!

Edit: I verified that BYOD does work on my system for up to 96kHz, where Proteus can't keep up at 96kHz. As you stated, multiples of 44.1k seem to work.  I will attempt to implement the same sample rate agnostic method used in BYOD. I think this should improve the sound as well when working with non - 44.1kHz.

**Comment by laurencebush** [comment]
I can verify that BYOD has a GuitarML module under drive and it seems to be working with high sample rates eliminating the aliasing.
